velox/docs/functions/presto/decimal.rst

```diff
@@ -127,10 +126,42 @@ difference in s1 and s.
 
 	p = p1 + s2 + max(0, s2 - s1)
 
-Like in Addition, the precision of the result may exceed 38. The choices are the
-same. Presto chooses to cap p at 38 and allow runtime errors.
+Let’s say a is a decimal(p1, s1) with unscaled value A and b is a decimal(p2, s2) with unscaled value B.
+
+::
+
+    a = A / 10^s1
+    b = B / 10^s2
+
+The result type precision and scale are:
+
+::
+
+	s = max(s1, s2)
+	p = p1 + s2 + max(0, s2 - s1)
+
+The final result:
+::
+
+    r = A * 10^(s + s2 - s1) / B
+
+To compute this value, we first rescale A using the rescale factor (s + s2 - s1), then divide by B and
+round to the nearest whole. This method works as long as the rescale factor doesn’t exceed 38. If s + s2 - s1
+exceeds 38 an error is raised.
 
-Velox implementation matches Presto.
+**Result Scale and rescale factor**
+
+Let's understand why result scale is :code:`max(s1,s2)`. This is crucial in computing rescaling factor (rf).
+Let's assume A= Decimal(10, 0) and B=Decimal(13, 13) and we are computing :code:`A/B`. The result scale
```

Comment:
`A = Decimal(10, 0) and B = Decimal(13, 13)`. Use code block?

Reply:
Done.